Update Inflation Page (#5479)
w3f/polkadot-wiki: docs/learn/learn-inflation.md

{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} is an inflationary token. On the
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} network, inflation is
{{ polkadot: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/polkadot/src/lib.rs#L576). :polkadot }}
{{ kusama: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/kusama/src/lib.rs#L535). :kusama }}Depending

> on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} supply locked and the ideal staking

Reviewer: ```suggestion
on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} supply staked and the ideal staking
```

rate (more about this below), part of the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}

> inflation goes to stakers and part to the [treasury](./learn-polkadot-opengov-treasury.md).

Reviewer: ```suggestion
inflation is distributed to the stakers and part to the [treasury](./learn-polkadot-opengov-treasury.md).
```

{{ kusama: [set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/kusama/src/lib.rs#L535). :kusama }}Depending
on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} supply locked and the ideal staking
rate (more about this below), part of the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
inflation goes to stakers and part to the [treasury](./learn-polkadot-opengov-treasury.md).

:::info

DOT went through [redenomination](./learn-redenomination.md) in 2020 that saw the DOT token supply
increase by 100 times.

The current token supply on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is
{{ polkadot: <RPC network="polkadot" path="query.balances.totalIssuance" defaultValue={12230666300429914781} filter="humanReadable"/> (Over 1.2 Billion DOT). :polkadot }}

> {{ kusama: <RPC network="kusama" path="query.balances.totalIssuance" defaultValue={12619256191792480093} filter="humanReadable"/> (Over 12 Million KSM). :kusama }}

Reviewer: <img width="811" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/eaa94dba-e8fe-49b8-ae72-e1ff23278b69">

Knowing the quantity of DOT or KSM by taking a glance at this number is tough (especially for those who are not from the ecosystem and who do not have any idea about the total issuance. That is why an approximate human readable number quantifying the supply was added. If you plan on making this accessible for diverse audience, either automate this in the human readable filter or keep the approximate values as is.

Author: re-added, will create a filter in another PR

{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} tokens idle with the current inflation model
as the tokens get diluted over time. Economics and game theory suggest that setting an ideal
inflation rate is essential for incentivizing the network participants as well as the growth of the
network, and any deviation from it can have adverse effects. Reducing the inflation rate could limit
growth, while increasing the inflation rate could break the incentive model of the token. Hence,
**token inflation rate is not a forever fixed value, and inflation can be updated in the future
through [on-chain governance](./learn-polkadot-opengov.md)** based on thorough tokenomics research.

## Inflation Model

The chart below shows the inflation model of the network. Depending on the number of staked tokens,
the distribution of the inflation to validators and nominators versus the treasury will change
dynamically to provide incentives to participate (or not participate) in staking.

There is a [dynamic _ideal staking rate_](#ideal-staking-rate) (in the figure set to 0.6 or 60%)
that the network tries to maintain. The inflation model will incentivize network participants to

> stake when the _system staking rate_ < _ideal staking rate_ and disincentivize staking when _system

Reviewer: At first glance, the word "system staking rate" is ambiguous to a new reader, making a reader think it is somehow dependent on the system or protocol. An apt name would be "current staking rate" or "staking rate in the current era" etc. If we already agreed on standardizing "system staking rate" in earlier discussions that I'm not aware of, you can ignore this suggestion.

```suggestion
stake when the _current staking rate_ < _ideal staking rate_ and disincentivize staking when _current
```

> staking rate_ > _ideal staking rate_. The goal is to have the system staking rate meet the ideal

Reviewer: ```suggestion
staking rate_ > _ideal staking rate_. The goal is to have the staking rate meet the ideal
```

> staking rate. The system staking rate would be the total amount staked over the total token supply,
> where the total amount staked is the stake of all validators and nominators on the network. The

Reviewer: Better to move this definition earlier in this section, before introducing the term.

```suggestion
staking rate. The current staking rate would be the total amount staked in the current era over the total token supply,
where the total amount staked is the stake of all validators and nominators on the network. The
```
